src/main/java/com/faforever/client/fx/PlatformServiceImpl.java

```diff
@@ -14,8 +16,15 @@
 
   private final HostServices hostServices;
 
+  private boolean isOnWindows;
+
+  @Value("${forgedAlliance.windowTitle}")
+  private String faWindowTitle;
```

Comment:
I agree with the change.
`PlatformServiceImpl` is instantiated a little differently than other beans (in Main.java), let's clean that up.
I think the result of `getHostServices()` should be added as a bean `hostServices` and the PlatformServiceImpl should get a @Service annotation and have `HostServices` injected.